### ISSUE TITLE
[imio/helm-plausible-analytics] mailer not working

### ISSUE THREAD

**Issue by callmewhatuwant** [IMIO/helm-plausible-analytics#18]
**Describe the bug**
The mailer does not work i always get : 
14:51:54.120 request_id=F90ypFH6BxjMTFAAAAYE [error] Failed to send e-mail

I have a plane.so pod that works perfect with the same gmail-user, password and port.

I just wanted to test it. 
Could there be a problem because I did add a random site to track and because of that I could not add the js snippet?

Does this effect my issue?: 
They added a new adapter maybe this is a problem.
I think the new one has another name: 
https://github.com/plausible/analytics/issues/3839

```
"Bamboo.Mua" ->
    config :plausible, Plausible.Mailer, adapter: Bamboo.Mua

    if relay = get_var_from_path_or_env(config_dir, "SMTP_HOST_ADDR") do
      port = get_int_from_path_or_env(config_dir, "SMTP_HOST_PORT", 25)
      username = get_var_from_path_or_env(config_dir, "SMTP_USER_NAME")
      password = get_var_from_path_or_env(config_dir, "SMTP_USER_PWD")

      config :plausible, Plausible.Mailer,
        auth: [username: username, password: password],
        relay: relay,
        port: port
    end
```

So i don't understand why it works with plane.so and not with plausible

**Version of Helm, Kubernetes and the plausible-analytics chart**:
helm:
version.BuildInfo{Version:"v3.15.2", GitCommit:"1a500d5625419a524fdae4b33de351cc4f58ec35", GitTreeState:"clean", GoVersion:"go1.22.4"}

Kubernetes:
Client Version: v1.29.5+k3s1

Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.29.5+k3s1

chart:
0.3.0

**What happened**:
error:
14:51:54.120 request_id=F90ypFH6BxjMTFAAAAYE [error] Failed to send e-mail

**What you expected to happen**:
I receive a email and it works

**How to reproduce it** (as minimally and precisely as possible):
create custom.yaml
change mailer
```
mailer:
  enabled: true  # Enable/Disable functionality
  email: "gmail-user"  # the email address of the email sender
  adapter: "Bamboo.SMTPAdapter"  # "Bamboo.SMTPAdapter", "Bamboo.MailgunAdapter", "Bamboo.MandrillAdapter", 
  smtp:
    host: "smtp.gmail.com"  # The host address of your smtp server.
    port: "465"  # The port of your smtp server.
    auth: true  # If SMTP auth is enabled.
    username: "gmail-user"  # The username/email in case SMTP auth is enabled.
    password: "my-app-password-from-gmail"  # The password in case SMTP auth is enabled.
    ssl: true  # If SSL is enabled for SMTP connection
    retries: "5"  # Number of retries to make until mailer gives up.
  mailgun:
    apiKey: ""
    domain: ""
    baseURI: ""
  postmarkApiKey: ""
  mandrillApiKey: ""
  sendgridApiKey: ""
```

**Anything else we need to know**:

Here are some information that help troubleshooting:

* if relevant, provide your `values.yaml` or the changes made to the default one (after removing sensitive information)
* the output of the folowing commands:
```
databaseURL: "customurl"
clickhouseDatabaseURL: "customurl"

baseURL: https://.....
listenIP: 0.0.0.0
secretKeyBase: ""  # is automaticly generated if left empty
totpVaultKey: "dsxvbn3jxDd16az2QpsX5B8O+llxjQ2SJE2i5Bzx38I="  # Should be updated! (TOTP_VAULT_KEY variable must exactly 32 bytes long)
disableRegistration: false  # Restricts registration of new users.
logFailedLoginAttempts: false  # Controls whether to log warnings about failed login attempts.

mailer:
  enabled: true  # Enable/Disable functionality
  email: "gmail-user"  # the email address of the email sender
  adapter: "Bamboo.SMTPAdapter"  # "Bamboo.SMTPAdapter", "Bamboo.MailgunAdapter", "Bamboo.MandrillAdapter", 
  smtp:
    host: "smtp.gmail.com"  # The host address of your smtp server.
    port: "465"  # The port of your smtp server.
    auth: true  # If SMTP auth is enabled.
    username: "gmail-user"  # The username/email in case SMTP auth is enabled.
    password: "my-app-password-from-gmail"  # The password in case SMTP auth is enabled.
    ssl: true  # If SSL is enabled for SMTP connection
    retries: "5"  # Number of retries to make until mailer gives up.
  mailgun:
    apiKey: ""
    domain: ""
    baseURI: ""
  postmarkApiKey: ""
  mandrillApiKey: ""
  sendgridApiKey: ""

## IP Geolocation
## Plausible use a free ip <-> country database to enrich analytics optionally
## you can provide a different database

extra_geolocation:
  enabled: false  # Enable/Disable
  maxmind:
    licenseKey: ""
    edition: ""
  geolite2CountryDB: ""
  geonamesSourceFile: ""

postgresql:
  enabled: true
  auth:
    enablePostgresUser: false
    username: plausible_user
    password: password
    database: plausible_db
  primary:
    containerSecurityContext:
      enabled: true
      seLinuxOptions: {}
      runAsUser: 50000
      runAsGroup: 50000
      runAsNonRoot: true
      privileged: false
      readOnlyRootFilesystem: true
      allowPrivilegeEscalation: false
      capabilities:
        drop: ["ALL"]
      seccompProfile:
        type: "RuntimeDefault"

    podSecurityContext:
      enabled: true
      fsGroupChangePolicy: Always
      sysctls: []
      supplementalGroups: []
      fsGroup: 50000

    persistence:
      enabled: true
      subPath: ""
      size: 8Gi
      storageClass: local-path-default-d8
      accessModes:
        - ReadWriteOnce
      annotations:
        storagepath: "plausible-database"
    persistentVolumeClaimRetentionPolicy:
      enabled: true
      whenScaled: Retain
      whenDeleted: Delete

clickhouse:
  enabled: false
```
I use another clickhouse chart thats why it is disabled an i have createt a custom storage resource with the ranchere
local storage provisioner: https://github.com/rancher/local-path-provisioner

Check if a pod is in error: 
```bash
kubectl get pod
```
NAME                                   READY   STATUS    RESTARTS   AGE
my-clickhouse-shard0-0                 1/1     Running   0          3h22m
plausible-analytics-postgresql-0       1/1     Running   0          19m
plausible-analytics-6489c7c7dc-9rzr5   1/1     Running   0          11m

Inspect the pod, check the "Events" section at the end for anything suspicious.

```bash
kubectl describe pod 
```
no errors

Get logs on a failed container inside the pod (here the `server` one):

```bash
kubectl logs pod_name
```
i have no failed containers

**Comment by callmewhatuwant** [comment]
I tested today again but changed the image you used (community)to: plausible/analytics version 2.0.0.
This image works.

I received an email.

Could you fix your chart that we can also use the community edition ? If this is possible.

**Comment by alexnuttinck** [comment]
@callmewhatuwant It has been fixed in the last release (0.3.2). `Bamboo.Mua` is now the default mailer adapter.
